README.md

```diff
@@ -300,7 +300,6 @@ modules with mixed Java and Kotlin code for Kotlin classes, though.
 #### Incremental Kotlin compilation breaks compiler plugins
 
 There are two bugs that affect the Anvil Kotlin compiler plugin:
-* [Incremental compilation breaks compiler plugins](https://youtrack.jetbrains.com/issue/KT-38570)
 * [AnalysisResult.RetryWithAdditionalRoots crashes during incremental compilation with java classes in classpath](https://youtrack.jetbrains.com/issue/KT-38576)
 
 The Gradle plugin implements workarounds for these bugs, so you shouldn't notice them. Side effects
```

Comment:
Can the line after this about incremental compilation be removed?

Reply:
> Side effects are that incremental Kotlin compilation is disabled for stub generating tasks (which don't run a full compilation before KAPT anyways).

This one? It's still true, we disable incremental compilation for stub generating tasks.

Reply:
Ah sorry, I somehow missed this morning that that part is staying